t/t-locks.sh

```diff
@@ -76,6 +76,34 @@ begin_test "list a single lock (SSH)"
 )
 end_test
 
+begin_test "list a single lock (SSH; git-lfs-transfer)"
+(
+  set -e
+
+  reponame="locks-list-ssh-pure"
+  setup_pure_ssh
```

Comment:
The `"batch transfers with ssh endpoint (git-lfs-transfer)"` test in `t/t-batch-transfer.sh` has this `setup_pure_ssh` line immediately after the initial `set -e` and separated with whitespace, which I liked because it helps visually distinguish this test from the one above it, so maybe it's worth doing the same here too?